views/bt_mouse.c

```diff
@@ -46,6 +46,7 @@ struct BtMouse {
 
 #define MOUSE_MOVE_SHORT 5
 #define MOUSE_MOVE_LONG 20
+#define MOUSE_SCROLL 20
```

Comment:
20 is too high (e.g. scrolls more than one page in Chrome), please set to 2 (also for USB).